scripts/globus/setup_collection_directory.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euf -o pipefail
```

Comment:
Might want to add a description of how this script should be used.

Reply:
Good catch, on it